lib/flow.js

```diff
@@ -2,8 +2,19 @@
 
 module.exports = (api) => {
 
+  api.metasync.flow = (
```

Comment:
@aqrln I updated `done` signature for `flow` by last commit from `(data)` to `(err, data)` but `done` signature for `compose` will be untouched (but deprecated) for old code support. Later I am going to make alternative implementation of `parallel` and `sequential` for `flow` with new `done` signature. And will change `compose` in `metasync@0.3`